src/core/js/views/notifyView.js

```diff
@@ -60,13 +66,19 @@ define([
         },
 
         render: function() {
-            var data = this.model.toJSON();
-            var template = Handlebars.templates['notify'];
+            var data, html;
+
+            if (this.customView) {
+                html = this.customView.$el;
```

Comment:
`this.customView.el` rather then `this.customView.$el` as it is passed into `jQuery.html()` ?